CHANGELOG.md

```diff
@@ -2,6 +2,18 @@
 
 ## UNRELEASED
 
+## v0.103.0
+
+:new: What's new:
+- Adding cosmos db kvstore driver (#5915)
```

Comment:
```suggestion
- Add Azure Cosmos DB kvstore driver (#5915)
```
(I like the convention of [using the imperative voice in commit lines](https://cbea.ms/git-commit/#imperative); we missed here, but at least the changelog can be imperative!)